docs/qcfractal/source/install.rst

```diff
@@ -10,12 +10,29 @@ You can update qcfractal using `conda <https://www.anaconda.com/download/>`_::
 
     conda install qcfractal -c conda-forge
 
-This installs qcfractal and the NumPy dependancy.
+This installs qcfractal and the NumPy dependency, but *not* any of the quantum
+chemistry codes nor the software to run run :term:`Queue Managers <Manager>`.
 
 The qcfractal package is maintained on the
 `conda-forge channel <https://conda-forge.github.io/>`_.
 
 
+Conda Pre-Created Environments
+++++++++++++++++++++++++++++++
+
+Fractal can also be installed through pre-configured environments you can pull through our Conda Channel::
+
+    conda env create qcarchive/{environment name}
+    conda activate {environment name}
+
+The environments are created from the YAML files hosted on the Anaconda Cloud, which then need to be activated
+to use. You can find all of the environments `here <https://anaconda.org/qcarchive>`_.
```

Comment:
https://anaconda.org/QCArchive/environments

The above link gives us room to write descriptions for each env.